modules/boost.detail/1.83.0/overlay/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
```

Comment:
Since this is a header only library we should add

```
    features = [
        "parse_headers",
    ],
```

Reply:
This would also require adding the flag `--process_headers_in_dependencies` to the presubmit config